install-cloud-sdk-build-task/README.md

```diff
@@ -10,6 +10,8 @@ and allows caching of Goolge Cloud SDK versions, but requires the  [VSTS Tool In
 
 ## Usage
 
+This task only works on build agents with PowerShell (windows build agents).
```

Comment:
What do you think about "This task only works on Windows build agents"?  We can run Powershell on Unix now.

Reply:
Done.